src/en/charms-constraints.md

```diff
@@ -31,12 +31,24 @@ resulting system's resources will exceed the constraint-defined minimum.
 However, if the cloud cannot satisfy a constraint at all then an error will be
 emitted and a machine will not be provisioned.
 
-When using the localhost cloud, constraints are ineffectual due the nature of
-this cloud's underlying technology (LXD), where each machine will, by default,
-have access to **all** of the LXD host's resources. Here, an exact hardware
-specification can be requested, but is done at the LXD level (see example
-below).
+### Constraints and the LXD cloud
 
+For the 'lxd' cloud type (`v.2.4.1`) things are different. With this cloud,
+constraints are interpreted as resource maximums as opposed to minimums. In the
+absence of constraints, a machine (container) will, by default, have access to
+**all** of the underlying system's (LXD host) resources.
+
+This cloud type also honours instance constraints where 
+[AWS instance type][aws-instance-types-kirkland] names are used (e.g.
+'t2.micro' maps to 1 CPU and 1 GiB of memory). When used in combination with
+specific CPU/MEM constraints the latter values will override corresponding
+instance type values.
+
+!!! Warning:
+    Due to the different meaning of constraints in the context of a LXD
+    cloud, care must be taken when using commands in which the cloud type is
+    not called out.
```

Comment:
cloud type is not called out? What are you warning of here?

Reply:
The fact that the given constraint in these example commands do entirely different things:

`juju deploy <charm> --constraints mem=4G`

and

`juju deploy <charm> --constraints mem=4G`

So one command is applied towards a LXD type cloud and one is applied towards a non-LXD type cloud. That is, the cloud is not "called out" (referenced).

Reply:
Right but since there's nothing in the command "called out". I can see saying something about "which cloud you're using" or something. But called out feels like I'm looking for a cloud type and it's not there.

Reply:
I'll reword.